x-pack/plugins/security_solution/server/lib/entity_analytics/risk_engine/types.ts

```diff
@@ -98,6 +98,16 @@ export interface DisableRiskEngineResponse {
   success: boolean;
 }
 
+export interface RiskEnginePrivilegesResponse {
```

Comment:
If this is shared between the client and server, it would be most appropriate in the `common` folder, e.g. `security_solution/common/entity_analytics/risk_engine/types.ts`